src/constants/earthEngineLayers/buildings_GOOGLE.js

```diff
@@ -0,0 +1,30 @@
+import i18n from '@dhis2/d2-i18n'
+import { EARTH_ENGINE_LAYER } from '../layers.js'
+
+export default {
+    layer: EARTH_ENGINE_LAYER,
+    format: 'FeatureCollection',
+    layerId: 'GOOGLE/Research/open-buildings/v1/polygons',
+    img: 'images/buildings.png',
+    datasetId: 'GOOGLE/Research/open-buildings/v1/polygons',
+    name: i18n.t('Building footprints'),
```

Comment:
Are you sure these i18n.t functions are valid JSON?

Reply:
Not valid JSON - we need to find another translation solution if we start to save these configs in a data store.

Reply:
You set this as resolved, but the json is still invalid, right? What about using constants, and then using a function to return the i18n string at runtime?

`name: BUILDING_FOOTPRINTS,`

```
const getEELayerName = name => {
    const nameMap ={
        BUILDING_FOOTPRINTS: i18n.t('Building footprints'),
        ...
    }

    return nameMap[name]
}
```

Reply:
I don't think this needs to be valid JSON as layer configs are still included in the app.